src/frontend/apps/e2e/__tests__/app-desk/footer.spec.ts

```diff
@@ -87,4 +87,31 @@ test.describe('Footer', () => {
       await expect(page).toHaveURL(url);
     });
   }
+
+  test('check if the app version is visible', async ({ page }) => {
+    await page.route('**/api/v1.0/config/', async (route) => {
+      const request = route.request();
+      if (request.method().includes('GET')) {
+        await route.fulfill({
+          json: {
+            RELEASE: '1.5.1',
+            LANGUAGES: [
+              ['en-us', 'English'],
+              ['fr-fr', 'French'],
+            ],
+            FEATURES: { TEAMS: false },
+          },
+        });
+      } else {
+        await route.continue();
+      }
+    });
```

Comment:
Actually it could have been interesting to not mock, but to intercept the response of `/api/v1.0/config/` to get the value of `RELEASE`, and then assert on the true backend value, here an interception example:
https://github.com/numerique-gouv/people/blob/579657afa412eb2ce65599de910543c79ac29747/src/frontend/apps/e2e/__tests__/app-desk/member-create.spec.ts#L33-L36